client/dashboard/sites/site-fields.tsx

```diff
@@ -0,0 +1,29 @@
+import { useQuery } from '@tanstack/react-query';
+import { useInView } from 'react-intersection-observer';
+import { siteUptimeQuery } from '../app/queries/site-uptime';
+import { TextBlur } from '../components/text-blur';
+import { getSiteUptime } from '../utils/site-uptime';
+import type { Site } from '../data/types';
+
+export function Uptime( { site }: { site: Site } ) {
+	const { ref, inView } = useInView( { triggerOnce: true, fallbackInView: true } );
+	const { data: uptime, isLoading } = useQuery( {
+		...siteUptimeQuery( site.ID, { period: 'week' } ),
+		enabled: site.is_wpcom_atomic && inView,
+		staleTime: 1000 * 60 * 60, // 1 hour
```

Comment:
Adding a stale time here otherwise the network request will happen every time the page changes. I tried with the default staleTime: 0 and setting refetchOnMount: false, but I still saw the network requests happening every time the page changes.